BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -569,6 +519,17 @@ fileprivate extension StoresTableViewController {
             .addBody("User navigated to \(store.storeName)\n\n\(store)")
             .withPriority(.medium)
             .send()
+        
+    }
+    
+    func makeNoteThatCellFailedToDequeue(cell: String) {
+        
+        LOG.error("Failed to dequeue \(cell)")
+        AromaClient.beginMessage(withTitle: "Failed to dequeue \(cell)")
```

Comment:
Might remove the `\(cell)` from the string, so that it shows up with the same title on Aroma.